test/root_finding_tests/root_finding_tests.jl

```diff
@@ -30,7 +30,8 @@ big_pi = @interval(pi)
 set_interval_precision(Float64)
 float_pi = @interval(pi)
 
-Base.⊆(a::Interval, b::Root) = a ⊆ b[1]   # the Root object has the interval in the first entry
+Base.⊆(a::Interval, b::Root) = a ⊆ b.interval   # the Root object has the interval in the first entry
```

Comment:
I think this functions should be defined in the package itself. I propose to move them to `root_finding/root_finding.jl`